### PR TITLE
feat: option to suppress value export

### PR DESCRIPTION
**Pull request by lunsdorf**
## Description
Implements options to disable value export, as mentioned in  #672.

## How Has This Been Tested?
Added unit test for option.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/processor/test/options.test.js [hunk 1]
```diff
@@ -146,6 +146,38 @@ describe("/processor.js", () => {
             });
         });
 
+        describe("exportValues", () => {
+            it("should not export @values by default", async () => {
```
Comment (tivac) [inline packages/processor/test/options.test.js]:
```suggestion
            it("should export @values by default", async () => {
```

Comment (lunsdorf) [reply packages/processor/test/options.test.js]: Fixed typo

**Comment by lunsdorf** [comment]
Nice! Thanks a lot for the the quick response and actually integrating it.

**Comment by tivac** [comment]
Released in [v25.1.0](https://github.com/tivac/modular-css/releases/tag/v25.1.0)

**Pull request merged**